### ISSUE TITLE
Rainloop ownCloud App 5.1.0 forgets all saved mailaccount passwords from ownCloud user settings

### ISSUE THREAD

**Issue by plastikschnitzer** [RainLoop/rainloop-webmail#1760]
**RainLoop version, browser, OS:**
Rainloop ownCloud App Version 5.1.0

**Expected behavior and actual behavior:**
I have added the mail account in the ownCloud settings before, so if I go to the Mail app, my account opens automatically.
After the update from 5.0.5 to Rainloop ownCloud app version 5.1.0, the password seems not to work any more, instead of the mailbox I get the login screen. 
If I go to the ownCloud settings and re-enter my password (my email address and asterisks in the password field are still there!) and save the data again, its working.

**Steps to reproduce the problem:**
Install ownCloud Rainloop app version 5.0.5
Add a mail account in the ownCloud user settings
Access your mail account (working)
Update to Rainloop app version 5.1.0
Try to access your mailbox (not working)

Is there a way to solve this? I have ownCloud installations with tons of mail accounts and adding all their user credentials manually would be a nightmare …

**Comment by deMattin** [comment]
Have a look here, maybe the same problem:
#1753 Update from 1.12.0 to 1.12.1 not using old data folder and setting

**Comment by plastikschnitzer** [comment]
No, thats not the case. The rainloop data folder and content stays the same. All rainloop admin panel settings are kept over the update but all users who saved their mailaccount in the ownCloud settings are gone. Where does rainloop save these settings?

**Comment by plastikschnitzer** [comment]
No, I only have one _data_ folder, no matter which version. No _data_blablabla folder in the former version. Very sure! I even made a file comparison and all files and folders are the same, only the content of rainloop-storage/INSTALLED and rainloop-storage/VERSION changes from 1.11.3 to 1.12.0 (as expected :-) )

Now I am re-entering tons of passwords in order to make it work … after that everything is back, all account settings, everything.

**Comment by plastikschnitzer** [comment]
I think I have a new trace: The problem does not occur when contacts are saved to sqlite Datebase. If I set it to mysql, the behavior described above occurs.

Any ideas why? It does not sound logical to me, but I could rebuild the error this way.
